src/Http/Request.php

```diff
@@ -89,11 +89,12 @@ public function getParameters()
 
     /**
      * @param string $name
+     * @param null $default
      * @return mixed
      */
-    public function getParameter(string $name)
+    public function getParameter(string $name, $default = null)
```

Comment:
type hint for $default ?

Reply:
it's mixed because it can get string or int
Ex: getParameter('id',0) or getParameter('name','')